sdk/identity/azure_identity/src/refresh_token.rs

```diff
@@ -71,7 +72,6 @@ pub struct RefreshTokenResponse {
     access_token: Secret,
     refresh_token: Secret,
 }
-azure_core::json_model!(RefreshTokenResponse);
```

Comment:
Rebase on `feature/track2`. There's at least one other one that needs to be updated.

Reply:
Come to think of it, should this PR just remove the old macros entirely?

Reply:
It does, but this usage was floating around still.